plasmapy/plasma/equilibria1d.py

```diff
@@ -0,0 +1,100 @@
+__all__ = ["HarrisSheet"]
+
+import astropy.constants as const
+import astropy.units as u
+import math
+import numpy as np
+from plasmapy.utils.decorators.validators import validate_quantities
+
+class HarrisSheet:
+
+    """
+    Define a Harris Sheet Equilibrium.
```

Comment:
Could you add a description of the coordinate systems?  i.e. that the magnetic field will be in the $\pm \hat{x}$ direction and the current density will be in the `\hat{z}` direction.